test/WebJobs.Script.Tests/EndToEndTestFixture.cs

```diff
@@ -125,6 +138,41 @@ private void DeleteEntities(string partition)
             }
         }
 
+        /// <summary>
+        /// Clean up files 
+        /// </summary>
+        /// <param name="connectionString">connection string pointing to SAAS connection
+        /// <remarks> Connection for DropBox is enabled if the <code>AzureWebJobsDropBox</code> environment variable is set.   
+        /// The format should be: <code>Endpoint={endpoint};Scheme={scheme};AccessToken={accesstoken}</code>
+        /// or to use the local file system the format should be: <code>UseLocalFileSystem=true;Path={path}</code>
+        /// </remarks>
+        /// </param>
+        /// <param name="throwOnMissingConnectionString">throw error if connection string is empty.</param>
+        public async Task CleanupApiHubTest(string connectionString, bool throwOnMissingConnectionString = true)
+        {
+            if (string.IsNullOrEmpty(connectionString))
+            {
+                if (throwOnMissingConnectionString)
+                {
+                    throw new ApplicationException("Missing AzureWebJobsDropBox environment variable.");
+                }
+
+                return;
+            }
+
+            string testBlob = "teste2e";
+            string apiHubFile = "teste2e/test.txt";
```

Comment:
Could we instead simply use a Guid file name for the test binding so there are never any collisions? Other tests are using that approach.

It really seems like we need to do that anyways, if internally on the team we're all sharing the same DropBox connection for testing. Otherwise concurrent test runs will be interfering with each other, right?